packages/react-reconciler/src/ReactDebugFiberPerf.js

```diff
@@ -332,14 +333,17 @@ export function startWorkLoopTimer(): void {
   }
 }
 
-export function stopWorkLoopTimer(): void {
+export function stopWorkLoopTimer(didInterrupt: boolean): void {
   if (enableUserTimingAPI) {
     if (!supportsUserTiming) {
       return;
     }
-    const warning = commitCountInCurrentWorkLoop > 1
-      ? 'There were cascading updates'
-      : null;
+    let warning = null;
+    if (didInterrupt) {
+      warning = 'Interrupted an in-progress update';
+    } else if (commitCountInCurrentWorkLoop > 1) {
+      warning = 'There were cascading updates';
+    }
```

Comment:
@gaearon Wasn't sure what to do here. Should we show both warnings? How should they be formatted?

Reply:
Maybe show more important one? There's not much screen space there.